frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.scss

```diff
@@ -34,3 +34,7 @@
       display: inline !important; //To override bootstrap help-block
   }
 }
+
+.ocs-storage-class-thickprovision__checkbox {
+  margin-bottom: var(--pf-global--spacer--sm) 0 !important; // To override checkbox class
+}
```

Comment:
checkbox class gives some extra margin-bottom, so it affects the description block